core/build.gradle

```diff
@@ -1,17 +1,14 @@
 plugins {
-    id 'com.novoda.bintray-release' version '0.8.0'
```

Comment:
The bintray-release plugin is not in the Gradle Plugins portal yet, so no need to add obscure setup (see changes in `settings.gradle`) when we can just apply the plugin as usual.
The plan here is to use composite builds (eg: in samples) so I want to remove all the possible cause of friction in that regard